ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/compliance.coverage.utils.tsx

```diff
@@ -112,8 +108,23 @@ export function getComplianceLabelGroupColor(
     return 'blue';
 }
 
+export function sortCheckStats(items: ComplianceCheckStatusCount[]): ComplianceCheckStatusCount[] {
+    const order: ComplianceCheckStatus[] = [
+        'PASS',
+        'FAIL',
+        'MANUAL',
+        'ERROR',
+        'INFO',
+        'NOT_APPLICABLE',
+        'INCONSISTENT',
+    ];
+    return [...items].sort((a, b) => {
+        return order.indexOf(a.status) - order.indexOf(b.status);
+    });
+}
+
 const statusIconTextMap: { [key in ComplianceCheckStatus]: ClusterStatusObject } = {
-    [ComplianceCheckStatusEnum.PASS]: {
+    PASS: {
```

Comment:
Commenting on the line above.

What are pro and con to replace type with `Record<ComplianceCheckStatus, ClusterStatusObject>`

https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type

Reply:
The index signature is technically more flexible but we don't need that flexibility here so `Record` probably makes more sense and is easier to read.